### ISSUE TITLE
Import files from Google Spreadsheet

### ISSUE THREAD

**Issue by javiavid** [divamgupta/obsidian-spreadsheets#1]
Thank you very much for the plugin, I was waiting for someone to do something like this.

Is it possible to somehow import a google spreadsheet document into the plugin?
I have tried copying and pasting, but it only pastes the values, not the formulas.

**Comment by divamgupta** [comment]
Yup, all that is in the plan. Depends on how much interest this plugin gets.

**Comment by spiritualgeek** [comment]
This is one plugin that I'll be happy to donate to if you give it love :)

**Comment by amatya-aditya** [comment]
Import and Export Function: https://github.com/ljcoder2015/obsidian-excel
